adapters/rubicon/rubicon.go

```diff
@@ -945,6 +943,58 @@ func updateUserExtWithIabAttribute(userExtRP *rubiconUserExt, data []openrtb2.Da
 	return nil
 }
 
+func updateSiteExtWithIabAttribute(siteExtRP *rubiconSiteExt, data []openrtb2.Data) error {
+	siteSegtaxes := []int{1, 2}
+	var segmentIdsToCopy = getSegmentIdsToCopy(data, siteSegtaxes)
+
+	siteExtRPTarget := make(map[string]interface{})
+
+	if siteExtRP.RP.Target != nil {
+		if err := json.Unmarshal(siteExtRP.RP.Target, &siteExtRPTarget); err != nil {
+			return &errortypes.BadInput{Message: err.Error()}
```

Comment:
This branch seems to not be covered by the tests

Reply:
Actually all test cases are in adapters/rubicon/rubicontest/exemplary/simple-video.json

Reply:
Yes. Both a json test or a coded test could have been used to address the coverage. We encourage json tests because they simplify the testing framework and we've improved the json test to detect race conditions. In the case of this particular lines of code. These have been changed in one of your recent commits and I believe the scenarios are well covered